src/widgetastic/widget.py

```diff
@@ -209,8 +209,9 @@ def __new__(cls, name, bases, attrs):
                 # handle read() specifics
                 new_attrs[key] = logged(log_result=True)(value)
             else:
-                # Do nothing
-                new_attrs[key] = value
+                # VersionPick resolution wrapper
+                new_attrs[key] = resolve_version_pick(key)(value)
```

Comment:
How do you detect that it is a method? What about class attributes? You probably need to check if it is a callable, or a function ... depends how exactly the class machinery wraps the function as a method :)
We also need to implement a blacklist for this feature, so you can omit methods on purpose.